src/main/java/com/chopshop166/chopshoplib/sensors/CSDutyCycleEncoder.java

```diff
@@ -33,4 +36,45 @@ public CSDutyCycleEncoder(DigitalSource source) {
         super(source);
     }
 
+    @Override
+    public double getAbsolutePosition() {
+        double distance = super.getDistance();
+        if (distance == -this.positionOffset) {
```

Comment:
Please add a comment explaining this check, and why it only matters when the distance exactly equals the offset?